### PR TITLE
Change to svg badge

### PR DESCRIPTION
**Pull request by stevemao**
svg looks better on retina screens

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,14 +33,19 @@
...
-    "mocha": "^2.0.1"
+    "grunt": "^0.4.5",
+    "grunt-cli": "^0.1.13",
+    "grunt-concurrent": "^1.0.0",
+    "grunt-contrib-connect": "^0.9.0",
+    "grunt-contrib-sass": "^0.9.2",
+    "grunt-contrib-watch": "^0.6.1",
+    "jshint": "^2.6.3"
+  },
+  "peerDependencies": {
```
Comment (shinnn) [inline package.json]: peerDependency will be deprecated. https://github.com/npm/npm/issues/6565#issuecomment-74971689

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# grunt-browser-sync [![NPM version](https://badge.fury.io/js/grunt-browser-sync.png)](http://badge.fury.io/js/grunt-browser-sync)
+# grunt-browser-sync [![NPM version](https://badge.fury.io/js/grunt-browser-sync.svg)](https://npmjs.org/package/grunt-browser-sync)
```
Comment (shinnn) [inline README.md]: `https://npmjs.org/package/grunt-browser-sync` -> `https://www.npmjs.com/package/grunt-browser-sync`

File: .travis.yml [hunk 3]
```diff
@@ -3,5 +3,6 @@ git:
   depth: 2
 language: node_js
 node_js:
+  - 'iojs'
```
Comment (shinnn) [inline .travis.yml]: No explicit quotes. `'iojs'` -> `iojs`

**Comment by shinnn** [comment]
@shakyShane I prefer [shields.io](http://shields.io/) badges rather than [fury.io](fury.io)'s. What do you think?

**Comment by shinnn** [comment]
@stevemao Looks good PR, overall. Thanks for taking care of CI.

File: .travis.yml (hunk 3 above)
Comment (stevemao) [inline .travis.yml]: Why is that? I see people using quotes everywhere

http://docs.travis-ci.com/user/languages/javascript-with-nodejs/

https://github.com/gruntjs/grunt-contrib-connect/blob/master/.travis.yml

Comment (shinnn) [inline .travis.yml]: For consistency in one file. `node_js` is not quoted.

[BrowserSync/grunt-browser-sync repo uses this style](https://github.com/BrowserSync/browser-sync/blob/master/.travis.yml).

Comment (stevemao) [inline .travis.yml]: That one doesn't actually look quite consistent. `'0.12'` and `'0.10'` are quoted but `iojs` is not. Would you consider changing that one? If not, I'll fix this one soon.

Comment (shinnn) [inline .travis.yml]: The reason why `0.12` and `0.10` are quoted is that they are regarded as `Number` without quotes. I think the current style is consistent in the way it does not use explicit quotes as possible.

But, certainly, I'm too opinionated. I didn't know [the Travis CI doc uses that style](http://docs.travis-ci.com/user/languages/javascript-with-nodejs/). OK, now I don't care about quotes.

Comment (stevemao) [inline .travis.yml]: Consistency is important :+1: Will change it soon :smile:

**Comment by stevemao** [comment]
Hi @shinnn , I have made the changes. Cheers!!

**Comment by shinnn** [comment]
:clap: 

Merging this PR. I'll switch [fury.io](https://github.com/BrowserSync/grunt-browser-sync/pull/fury.io) to [shields.io](http://shields.io/) after we reach the concensus on https://github.com/BrowserSync/grunt-browser-sync/pull/96#issuecomment-77096280.

**Comment by shinnn** [comment]
@stevemao Thanks for your great work!

**Comment by stevemao** [comment]
No problem :smiley:

**Comment by shakyShane** [comment]
@shinnn yep, shields.io all the way.

Would be good to keep styles exactly as main repo 

shields

**Pull request merged**